blueocean-i18n/src/main/java/io/jenkins/blueocean/i18n/BlueI18n.java

```diff
@@ -136,12 +136,19 @@ public HttpResponse doDynamic(StaplerRequest request) {
     private JSONObject getBundle(BundleParams bundleParams, Locale locale) {
         PluginWrapper plugin = bundleParams.getPlugin();
```

Comment:
So if `BundleParams` supported the notion of an undefined plugin, then I think we could change the JS code to send that instead of "dummy".